docs/03-TechnicalInfrastructure.md

```diff
@@ -90,10 +90,10 @@ Jiv images produced by the imaging insertion pipeline are organized per
 
 #### The `logs` directory
 
-The logs of the scripts are created under `data/logs` in `/data/project`.
+The logs of the scripts are created under `data/logs` in `/data/$PROJECT`.
     
 ```
-## Content of the /data/project/data/logs directory
+## Content of the /data/$PROJECT/data/logs directory
 .
 |__ TarLoad.log
```

Comment:
Is the `TarLoad.log` a directory within `/data/$PROJECT/data/logs/`? If so, my `/data/$PROJECT/data/logs` is not structured as such. Currently, I just have the `TarLoad-X-XX-XXXXXX.log` files present. I understand that the other folders are created when other pipelines are executed, but should be outdented. Ex.
```
## Content of the /data/$PROJECT/data/logs directory
.
|__ TarLoad.log
|__ DTIPrep_pipeline
|   |__ DTI_QC`date`.log
|   |__ DTI_QC`date`.log
|__ DTIPrep_register
|   |__ DTIregister`date`.log
|   |__ DTIregister`date`.log
|__ registerProcessed
    |__ registerProcessed`date`.log
    |__ registerProcessed`date`.log
```

Reply:
agreed. I made the changes but I am roping @cmadjar here so she can undo and/or add changes on top of my changes.